compiler/runtime/OMRCodeCache.cpp

```diff
@@ -190,9 +186,7 @@ OMR::CodeCache::unreserve()
 void
 OMR::CodeCache::writeMethodHeader(void *freeBlock, size_t size, bool isCold)
    {
-#if defined(OSX) && defined(AARCH64)
-   pthread_jit_write_protect_np(0);
-#endif
+	omrthread_jit_write_protect_disable();
```

Comment:
Please fix the indentation of this line.

Reply:
Fixed in 94fa01869fd934edabbab476153e22f0ff4ccff7.